docs/stdlib/array.rst

```diff
@@ -261,17 +260,17 @@ Reference
 
     :index: set array unpack
 
-    Return array elements as a set.
-
-    .. note::
-
-        The ordering of the returned set is not guaranteed.
+    Returns the elements of an array as a set:
 
     .. code-block:: edgeql-repl
 
         db> select array_unpack([2, 3, 5]);
         {3, 2, 5}
 
+    .. note::
```

Comment:
Completely missed this. This change should be reverted because the example above illustrates the information referenced in the note. We should give the note first to give the context beforehand.

Once you revert it, you will also want to remove the colon from the one-liner.

Reply:
I believe I did this as note admonitions were sometimes split between the top and bottom of it. In hindsight we should remove the admonition in whole and have it as a separate paragraph, it's objectively a fundamental aspect of the function now that I see it.